crates/avalanche-types/Cargo.toml

```diff
@@ -153,7 +153,6 @@ message = [
 subnet = [
     "futures",
     "http",
-    "jsonrpc-core",
     "num-derive",
```

Comment:
If we're removing `jsonrpc-core` shouldn't we add the new crate?

Reply:
jsonrpc-core was pulled in so they could use `jsonrpc-core::futures`, since this isn't now needed doing it would be unnecessary